### PR TITLE
Feature: use Afterpay.locale to set brand assets

### PR DESCRIPTION
**Pull request by ScottAntonacAP**
## Summary of Changes

- Create a Brands class that returns resources (strings and drawables) based on locale
- Create an Afterpay.brand internal var with type of Brands class
- Replace all references to assets that contain brands (buttons, badges, service name etc) with appropriate Afterpay.brands reference
- Update string files where needed
- Move drawables from localised directory to primary directory (drawable-en-rGB to drawable)

## Items of Note

Reason for this change is the Afterpay and Clearpay brands should display based on the locale set by the merchant config rather than OS locale.

### CODE REVIEW

File: afterpay/src/main/kotlin/com/afterpay/android/internal/Brands.kt [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package com.afterpay.android.internal
+
+import com.afterpay.android.R
+import java.util.Locale
+
+class Brands(var locale: Locale) {
```
Comment (k-obrien) [inline afterpay/src/main/kotlin/com/afterpay/android/internal/Brands.kt:6]: I'd be more inclined to go for an enum in this case. Probably something like:
```kotlin
private val brandLocales = mapOf(
    setOf(AUSTRALIA, CANADA, NEW_ZEALAND, US) to AFTERPAY,
    setOf(UK) to CLEARPAY,
)

internal enum class Brand(
    @StringRes val title: Int,
    @StringRes val description: Int,
    @DrawableRes val badgeForeground: Int,
    @DrawableRes val payNowButtonForeground: Int,
    @DrawableRes val buyNowButtonForeground: Int,
    @DrawableRes val checkoutButtonForeground: Int,
    @DrawableRes val placeOrderButtonForeground: Int
) {

    AFTERPAY(
        title = R.string.afterpay_service_name,
        description = R.string.afterpay_service_name_description,
        badgeForeground = R.drawable.afterpay_badge_fg,
        payNowButtonForeground = R.drawable.afterpay_button_pay_now_fg,
        buyNowButtonForeground = R.drawable.afterpay_button_buy_now_fg,
        checkoutButtonForeground = R.drawable.afterpay_button_checkout_fg,
        placeOrderButtonForeground = R.drawable.afterpay_button_place_order_fg
    ),

    CLEARPAY(
        title = R.string.clearpay_service_name,
        description = R.string.clearpay_service_name_description,
        badgeForeground = R.drawable.clearpay_badge_fg,
        payNowButtonForeground = R.drawable.clearpay_button_pay_now_fg,
        buyNowButtonForeground = R.drawable.clearpay_button_buy_now_fg,
        checkoutButtonForeground = R.drawable.clearpay_button_checkout_fg,
        placeOrderButtonForeground = R.drawable.clearpay_button_place_order_fg
    );

    companion object {

        fun forLocale(locale: Locale): Brand =
            brandLocales.entries.find { locale in it.key }?.value ?: AFTERPAY
    }
}
```

File: afterpay/src/main/kotlin/com/afterpay/android/internal/Locales.kt [hunk 2]
```diff
@@ -11,4 +11,5 @@ internal object Locales {
     val US: Locale = Locale.US
 
     val validSet = setOf(AUSTRALIA, CANADA, UK, NEW_ZEALAND, US)
+    val clearpayBrands = setOf(UK)
```
Comment (k-obrien) [inline afterpay/src/main/kotlin/com/afterpay/android/internal/Locales.kt:14]: And remove this in favour of `brandLocales` above.

**Review by k-obrien — CHANGES_REQUESTED** [review]
I've made a couple of suggestions which I think improve encapsulation and enforce the need for passing resource IDs instead of random `Int`s.

File: afterpay/src/main/kotlin/com/afterpay/android/internal/Brands.kt (hunk 1 above)
Comment (ScottAntonacAP) [reply afterpay/src/main/kotlin/com/afterpay/android/internal/Brands.kt]: Agreed, that's much cleaner. Addressed in 380e8eb

File: afterpay/src/main/kotlin/com/afterpay/android/Afterpay.kt [hunk 3]
```diff
@@ -27,6 +28,10 @@ object Afterpay {
     internal val locale: Locale
         get() = configuration?.locale ?: Locales.US
 
+    // TODO: remove this
```
Comment (k-obrien) [inline afterpay/src/main/kotlin/com/afterpay/android/Afterpay.kt:31]: Why the TODO?

**Review by k-obrien — APPROVED** [review]
Couple of minor things, but looks good!

File: afterpay/src/main/kotlin/com/afterpay/android/Afterpay.kt (hunk 3 above)
Comment (ScottAntonacAP) [reply afterpay/src/main/kotlin/com/afterpay/android/Afterpay.kt]: Thanks for the pickup, thought i'd removed it before pushing. Removed in 368e21f

**Pull request merged**